book/src/integrations/oauth2.md

```diff
@@ -200,32 +200,48 @@ kanidm system oauth2 reset-secrets
 Each resource server has unique signing keys and access secrets, so this is limited to each resource
 server.
 
-## Extended Options for Legacy Clients
+## Custom Claim Maps
 
-Not all resource servers support modern standards like PKCE or ECDSA. In these situations it may be
-necessary to disable these on a per-resource server basis. Disabling these on one resource server
-will not affect others.
+Some OIDC clients may consume custom claims from an id token for access control or other policy
+decisions. Each custom claim is a key:values set, where there can be many values associated to a
+claim name. Different applications may expect these values to be formatted (joined) in different
+ways.
 
-<!-- deno-fmt-ignore-start -->
+Claim values are mapped based on membership to groups. When an account is a member of multiple
+groups that would recieve the same claim, the values of these maps are merged.
 
-{{#template ../templates/kani-warning.md
-imagepath=../images
-title=WARNING
-text=Changing these settings MAY have serious consequences on the security of your resource server. You should avoid changing these if at all possible!
-}}
+To create or update a claim map on a client:
 
-<!-- deno-fmt-ignore-end -->
+```
+kanidm system oauth2 update-claim-map <name> <claim_name> <kanidm_group_name> [values]...
+kanidm system oauth2 update-claim-map nextcloud account_role nextcloud_admins admin
```

Comment:
Can this be extended to show multiple claims just so it's clear, I assume they're space-delimited on the CLI?

```suggestion
kanidm system oauth2 update-claim-map nextcloud account_role nextcloud_admins admin secondclaim
```